enterprise/coderd/groups.go

```diff
@@ -420,3 +420,41 @@ func (api *API) groups(rw http.ResponseWriter, r *http.Request) {
 
 	httpapi.Write(ctx, rw, http.StatusOK, resp)
 }
+
+// @Summary Get reduced groups for a user in an organization
```

Comment:
```suggestion
// @Summary Get reduced groups by user and organization
```

Reply:
Thanks! The route was deleted, so no changes were necessary.